README.md

```diff
@@ -1,7 +1,5 @@
 # Postee
 
-![Postee](postee.jpeg)
-
```

Comment:
I'm using this logo for UI (for login screen). Should I remove it as well?

Reply:
No we'll first create the new logo then I'll open a PR that replaces.